channel_setup/delete_channels.sh

```diff
@@ -1,5 +1,24 @@
 #!/bin/bash
 
+if [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "NAME"
+  echo "  delete_channels - Delete numeracy channels"
+  echo
+  echo "SYNOPSIS"
+  echo "  ./delete_channels [CHANNEL_ID]"
+  echo
+  echo "DESCRIPTION"
+  echo "  With no CHANNEL_ID supplied, the script deletes all numeracy channels from a device"
```

Comment:
The script actually does not take in any arguments. It deletes all numeracy channels on the device